core/src/tile/tile.h

```diff
@@ -51,7 +49,7 @@ class Tile {
     float getScale() const { return m_scale; }
 
     /* Returns the reciprocal of <getScale()> */
-    float getInverseScale() const { return m_inverseScale; }
+    float getInverseScale() const { return 1.0f / m_scale; }
```

Comment:
I agree with avoiding division where possible for values that are used in a loop. But we are discussing here about avoiding one division per feature. In this case my rule of thumb would say to rather keep the struct lean to fit into one cacheline. 
In any case the effect is likely to be below a measurable threshold.